cp3pt0-deployment/migrate_tenant.sh

```diff
@@ -210,6 +216,10 @@ function pre_req() {
         success "oc command logged in as ${user}"
     fi
 
+    if [[ $LICENSE_ACCEPT != 1 ]]; then
+        error "License not accepted. Rerun script with --license-accept flag set. See https://ibm.biz/integration-licenses for more details"
```

Comment:
```suggestion
        warning "License not accepted. Rerun script with --license-accept flag set. See https://ibm.biz/integration-licenses for more details"
```

Let's just use warning since license-accept does not break operator functionality. Same for other files